README.md

```diff
@@ -252,10 +259,14 @@ tox -e py36-docker -- tests/test_google_analytics_provider.py
 
 Check out the [Docker README](/docker/README.md) in the `/docker` directory for in-depth information on optionally running and developing the Circulation Manager locally with Docker, or for deploying the Circulation Manager with Docker.
 
+## Debugging
+
+If you are served an error message on the admin home screen, you may need to run `npm install` in the api/admin directory of the circulation repo. If you're running the application locally, you can also try running the same command in the root directory of the circulation-web repo.
```

Comment:
You should add at the end that the latter command will work if the circulation-web and circulation repos are linked through `npm link`.

Reply:
Done!